src/boat_simulator/boat_simulator/nodes/physics_engine/physics_engine_node.py

```diff
@@ -110,9 +111,16 @@ def __init_private_attributes(self):
         self.__rudder_angle = 0
         self.__sail_trim_tab_angle = 0
         self.__desired_heading = None
-        self.__boat_state = BoatState(
-            0.5, 1, np.array([[0.5, 0.5, 0.5], [0.0, 0.5, 0.5], [0.0, 0.0, 0.5]], dtype=np.float32)
-        )
+        self.__boat_state = {
+            "sail_lift_coeffs": Constants.BOAT_PROPERTIES.sail_lift_coeffs,
+            "sail_drag_coeffs": Constants.BOAT_PROPERTIES.sail_drag_coeffs,
+            "sail_areas": Constants.BOAT_PROPERTIES.sail_areas,
+            "rudder_drag_coeffs": Constants.BOAT_PROPERTIES.rudder_drag_coeffs,
+            "rudder_areas": Constants.BOAT_PROPERTIES.rudder_areas,
+            "sail_dist": Constants.BOAT_PROPERTIES.sail_dist,
+            "rudder_dist": Constants.BOAT_PROPERTIES.rudder_dist,
+            "hull_drag_factor": Constants.BOAT_PROPERTIES.hull_drag_factor,
+        }
```

Comment:
Please continue using the `BoatState` class here. Update the second and third parameters to read from the constants file instead of the previous hardcoded values. The first parameter can remain as 0.5, but the second parameter should be the mass and the third should be the inertia, both of which should be sourced from the constants file.

Reply:
Sorry for the oversight—I forgot to ask you to add mass and inertia to the constants file before making this update. Since this PR is already quite lengthy, please revert to the original `self.__boat_state = BoatState(...)` initialization with the hardcoded values.